src/ResourceManager/Insights/Commands.Insights/help/Add-AzureRmLogProfile.md

```diff
@@ -8,7 +8,9 @@ schema: 2.0.0
 # Add-AzureRmLogProfile
 
 ## SYNOPSIS
-Creates a log profile.
+Creates a new activity log profile. This profile is used to either archive the activity log to a storage account or stream it to an Azure event hub. Only standard storage account (premium storage account is not supported) is supported. It could either be of type ARM or Classic. If it's logged to a storage account, the cost of storing the activity log is billed at normal standard storage rates. 
```

Comment:
Should also note that "If it is streamed to an event hub, standard event hub pricing is in place."